doc/nrf/ug_thingy53.rst

```diff
@@ -302,53 +275,151 @@ See `Updating Nordic Thingy:53 firmware`_ for the detailed procedures on how to
+The TX gain can be configured with the :kconfig:`CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB` Kconfig option to select between +10 dBM or +20 dBM gain.
+To use the **ANT2** antenna, disable the :kconfig:`CONFIG_MPSL_FEM` Kconfig option.
+
+.. note::
+   Transmitting wth TX output power above +10 dBM is not permitted in some regions.
+   See the `Nordic Thingy:53 Hardware`_ documentation for the applicable regulations in your region before changing the configuration.
+
+.. _thingy53_app_fota_smp:
+
+FOTA and SMP
+============
+
+FOTA updates on the Nordic Thingy:53 are supported using MCUmanager's Simple Management Protocol (SMP) over Bluetooth.
+The application acts as a GATT server and allows the cibbected Bluetooth Central to perform the firmware update for both application and network core.
+The Bluetooth configuration is updated to allow quick DFU data trasfer.
+
+The application supports SMP handlers related to:
+
+* Image management.
+* Operating System (OS) management -- used to reboot the device after the firmware upload is complete.
+* Erasing settings partition -- used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
+
+To enable the MCUmanager library and handling of SMP commands over Bluetooth, use :kconfig:`CONFIG_MCUMGR` and :kconfig:`CONFIG_MCUMGR_SMP_BT`.
+Also, disable encryption and authentication requirement for the Bluetooth SMP transport using :kconfig:`CONFIG_MCUMGR_SMP_BT_AUTHEN`.
+
+The MCUmanager handlers for image and OS management must be enabled.
+To do this, use :kconfig:`CONFIG_MCUMGR_CMD_IMG_MGMT` and :kconfig:`CONFIG_MCUMGR_CMD_OS_MGMT`.
+
+The MCUmanager processing of basic Zephyr group commands and storage erase commands must also be enabled.
+This is done using both :kconfig:`CONFIG_MCUMGR_GRP_ZEPHYR_BASIC` and :kconfig:`CONFIG_MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE`.
+
+See :file:`thing53_nrf5340_cpuapp.conf` in a compatible sample for how this is done.
+
+In addition, the application must call :c:func:`os_mgmt_register_group()` and :c:func:`img_mgmt_register_group()` to register the image and OS management command handlers.
+The application must also call :c:func:`smp_bt_register()` to register the SMP Bluetooth Service.
+
+See the :file:`thingy53.c` source file in a compatible sample for how this is done.
+
+.. _thingy53_app_external_flash:
+
+External flash
+==============
+
+During a FOTA update, there might not be enough available flash storage to store the existing application and network core images, so the incoming images must be stored in external flash storage.
+This means that a sample compatible with the Nordic Thingy:53 must enable the external flash using the QSPI driver.
+To do this, e by enabling :kconfig:`CONFIG_NORDIC_QSPI_NOR`, and setting :kconfig:`CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE` to ``4096`` and :kconfig:`CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE` to ``16``.
```

Comment:
"To do this, e by enabling.." <-- it looks like something happened to the second part of this sentence.

Reply:
This sentence used to be part of the previous sentence, and apparently my conversion work was interrupted. Fixed.